src/utils/error.ts

```diff
@@ -0,0 +1,11 @@
+export const isAlreadyExists = (err: Error, resource: string): boolean => {
+  const reg = new RegExp(`${resource} \"(.*)\" already exists`)
+  return reg.test(err.message)
+}
+
+export const resourceHash = (err: Error, resource: string): string => {
+  const reg = new RegExp(`${resource} \"(.*)\" already exists`)
```

Comment:
This should be done using the gRPC Error code... This is not an small fix as the Engine needs to be updated. But using a regex is a hack.

Reply:
definitely agree but unfortunately, for now, the engine doesn't return proper error code.